pkg/interop/native/policy/policy.go

```diff
@@ -91,3 +92,9 @@ func GetMillisecondsPerBlock() int {
 func SetMillisecondsPerBlock(value int) {
 	neogointernal.CallWithTokenNoRet(Hash, "setMillisecondsPerBlock", int(contract.States|contract.AllowNotify), value)
 }
+
+// GetBlockedAccounts represents `getBlockedAccounts` method of Policy native contract.
+// Note that this method is available starting from [config.HFFaun] hardfork.
+func GetBlockedAccounts() iterator.Iterator {
```

Comment:
Add test to pkg/compiler/interop_test.go, take TestStorageIterator_Value as an example.